hw1-app/src/App.js

```diff
@@ -1,25 +1,43 @@
 import logo from './logo.svg';
 import './App.css';
 
+import Button from './components/Button';
+import Counter from './components/Counter';
+import SearchBar from './components/SearchBar';
+import View from './components/View';
+
 function App() {
   return (
     <div className="App">
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
         <p>
-          Edit <code>src/App.js</code> and save to reload.
+          HW 1 Oleg Letianin.
         </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+          <p>
+            Learn React - components
+          </p>
+          <View children={[<SearchBar Name="Search"/>]}></View>          
+          <Counter text="0" Id="counter"></Counter>
+          <View children={[
+            <Button sign="+" handler={UpdateCount}></Button>,
+            <Button sign="-" handler={UpdateCount}></Button>
+          ]}>
+          </View>
       </header>
     </div>
   );
 }
 
+function UpdateCount(val) {
+  const element = document.getElementById("counter");
+  if(element) {
+    let value = parseInt(element.innerHTML);
+    if(!isNaN(value)) {
```

Comment:
We will talk about this function in React way of thinking and adopting JSX syntax.